js/hitbtc2.js

```diff
@@ -943,6 +943,27 @@ module.exports = class hitbtc2 extends hitbtc {
         //         address: '0xd53ed559a6d963af7cb3f3fcd0e7ca499054db8b',
         //     }
         //
+        //     {
+        //       "id": "4d241e54-6546-496c-b97c-52178938179d",
```

Comment:
This comment seems to be duplicating one of the above, no?

Reply:
Just wanted to show examples of `bankToExchange` and `exchangeToBank` which are internal transfers on the exchange.

Reply:
Yep, I mean, those are included there (a few lines above) already, no? )